src/coreclr/src/hosts/inc/coreclrhost.h

```diff
@@ -121,4 +123,11 @@ CORECLR_HOSTING_API(coreclr_execute_assembly,
 
 #undef CORECLR_HOSTING_API
 
+//
+// Callback types used by the hosts
+//
+typedef bool(BundleProbeFn)(const char* path, int64_t* offset, int64_t* size);
```

Comment:
These type definitions should include the CORECLR_CALLING_CONVENTION, otherwise on x86 Windows, a wrong calling convention might get used depending on what calling convention the host has set as default.